client/src/components/SolutionViewer/ProjectModal.tsx

```diff
@@ -54,7 +53,6 @@ const ProjectModal = props => {
   );
```

Comment:
I can't reach the line because of GitHub restrictions, but above here on line 47:

Replace
```tsx
        <SolutionViewer challengeFiles={challengeFiles} solution={solution} />
```
with
```tsx
        <SolutionViewer
          challengeFiles={challengeFiles}
          solution={String(solution)}
        />
```

Going to cc @ojeytonwilliams on this - it's a cheaty approach to address the type def, but might be better to re-evaluate the type def?

Reply:
![image](https://user-images.githubusercontent.com/63889819/136032414-1c17f054-22e8-4bd1-9f3c-0bf13878effc.png)

Looks like this hasn't been changed yet.